[2024-07-18T19:00:34.589+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-18T19:00:34.612+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T19:00:33.224079+00:00 [queued]>
[2024-07-18T19:00:34.620+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T19:00:33.224079+00:00 [queued]>
[2024-07-18T19:00:34.621+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-18T19:00:34.632+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): start_process> on 2024-07-18 19:00:33.224079+00:00
[2024-07-18T19:00:34.639+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'run_fill_procedure', 'start_process', 'manual__2024-07-18T19:00:33.224079+00:00', '--job-id', '480', '--raw', '--subdir', 'DAGS_FOLDER/create_and_fill_proc.py', '--cfg-path', '/tmp/tmpk15k3s59']
[2024-07-18T19:00:34.641+0000] {standard_task_runner.py:91} INFO - Job 480: Subtask start_process
[2024-07-18T19:00:34.647+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=11881) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-18T19:00:34.648+0000] {standard_task_runner.py:63} INFO - Started process 11882 to run task
[2024-07-18T19:00:34.678+0000] {task_command.py:426} INFO - Running <TaskInstance: run_fill_procedure.start_process manual__2024-07-18T19:00:33.224079+00:00 [running]> on host 7c7e02e57379
[2024-07-18T19:00:34.754+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_fill_procedure' AIRFLOW_CTX_TASK_ID='start_process' AIRFLOW_CTX_EXECUTION_DATE='2024-07-18T19:00:33.224079+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-18T19:00:33.224079+00:00'
[2024-07-18T19:00:34.755+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-18T19:00:34.774+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-18T19:00:39.800+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-18T19:00:39.801+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-18T19:00:39.812+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=run_fill_procedure, task_id=start_process, run_id=manual__2024-07-18T19:00:33.224079+00:00, execution_date=20240718T190033, start_date=20240718T190034, end_date=20240718T190039
[2024-07-18T19:00:39.832+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-18T19:00:39.844+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-07-18T19:00:39.860+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-18T19:00:39.863+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
